### PR TITLE
2031 api display overlay on non active map

### PR DESCRIPTION
**Pull request by ramtob**

### CODE REVIEW

File: src/app/@ansyn/ansyn/api/ansyn-api.service.ts [hunk 1]
```diff
@@ -88,6 +89,11 @@ export class AnsynApi {
 		})
 	);
 
+	onSetLayoutSuccess$: Observable<any> = this.actions$.pipe(
+		ofType(CoreActionTypes.SET_LAYOUT_SUCCESS),
+		tap(() => console.log(1))
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/api/ansyn-api.service.ts]: ?

File: src/app/@ansyn/ansyn/api/ansyn-api.service.ts [hunk 2]
```diff
@@ -118,6 +124,13 @@ export class AnsynApi {
 		this.store.dispatch(new DisplayOverlayAction({ overlay, mapId: this.activeMapId, forceFirstDisplay: true }));
 	}
 
+	displayOverlayOnMap(overlay: IOverlay, mapNo: number): void {
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/api/ansyn-api.service.ts]: make mapNo optional and if undefined display on active map (remove the other function)
mapNo -> mapNumber

File: src/app/@ansyn/ansyn/api/ansyn-api.service.ts [hunk 3]
```diff
@@ -118,6 +124,13 @@ export class AnsynApi {
...
+		if (mapNo > this.mapsList.length - 1) {
+			return
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/api/ansyn-api.service.ts]: return;

File: src/app/@ansyn/ansyn/api/ansyn-api.service.ts (hunk 2 above)
Comment (ramtob) [reply src/app/@ansyn/ansyn/api/ansyn-api.service.ts]: done

File: src/app/@ansyn/ansyn/api/ansyn-api.service.ts (hunk 3 above)
Comment (ramtob) [reply src/app/@ansyn/ansyn/api/ansyn-api.service.ts]: done differently

**Pull request merged**